tests/unit/test_tool.py

```diff
@@ -712,10 +711,53 @@ def test_insertion(self):
         self.assertEqual(False, bpps[0].opposing_strands)
 
 
+class TestStrelka(unittest.TestCase):
```

Comment:
can you add an SNV test here as well to show that we are dropping them?

Reply:
sure, but I was assuming we would only indels from Strelka as input